spec/services/licence_transaction/import_industry_sectors_spec.rb

```diff
@@ -1,21 +1,21 @@
 require "spec_helper"
 
 RSpec.describe LicenceTransaction::ImportIndustrySectors do
```

Comment:
I think we're now testing the wrong method (should be `call`) and as a result we're missing some tests for the features that have been added (reading / writing to JSON file). If we find it useful to have `imported_json_data` as its own public method then we should probably test both.

Reply:
You caught me! I'll have a bit of refactor and find a nice way to test the JSON file.